pkgs/os-specific/linux/bpfmon/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, lib, libpcap, yascreen }:
+
+stdenv.mkDerivation rec {
+  pname = "bpfmon";
+  version = "2.50";
+
+  src = fetchFromGitHub {
+    owner = "bbonev";
+    repo = "bpfmon";
+    rev = "v${version}";
+    sha256 = "sha256-x4EuGZBtg45bD9q1B/6KwjDRXXeRsdFmRllREsech+E=";
+  };
+
+  buildInputs = [ libpcap yascreen ];
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  meta = with lib; {
+    description = "BPF based visual packet rate monitor";
+    homepage = "https://github.com/bbonev/bpfmon";
+    maintainers = [ maintainers.arezvov ];
+    license = licenses.gpl2Only;
```

https://github.com/bbonev/bpfmon/blob/v2.50/bpfmon.c#L4
https://github.com/bbonev/bpfmon/blob/v2.50/Makefile#L3
```suggestion
    license = licenses.gpl2Plus;
```